src/metabase/driver/materialize.clj

```diff
@@ -3,29 +3,71 @@
   (:require [clojure
```

Comment:
Is it correct to say they we use exactly the Postgres Driver and disable all the troublemaker options (`set-timezone`, `datetime-diff`, etc.)?

Reply:
Yep exactly! We first register the Materialize driver and define Postgres as the parent driver, eg:

```
(driver/register! :materialize, :parent :postgres)
```

It is very similar to how the [Redshift](https://github.com/metabase/metabase/blob/master/modules/drivers/redshift/src/metabase/driver/redshift.clj) driver is setup as well.